outputs.tf

```
@@ -33,6 +33,16 @@ output "cf_identity_iam_arn" {
   description = "CloudFront Origin Access Identity IAM ARN"
 }
 
+output "cf_origin_groups" {
+  value       = try(flatten(aws_cloudfront_distribution.default.*.origin_group), [])
+  description = "List of Origin Groups in the CloudFront distribution."
+}
+
+output "cf_origin_ids" {
+  value       = try([for origin in aws_cloudfront_distribution.default[0].origin : origin.origin_id], [])
```

Could this be simplified ?

```suggestion
  value       = try(aws_cloudfront_distribution.default[0].origin.*.origin_id, [])
```

Strange, I cannot see the description of this attribute on the [aws_cloudfront_distribution](https://registry.terraform.io/providers/hashicorp/aws/latest/docs/resources/cloudfront_distribution#attributes-reference) resource page.